nestkernel/ring_buffer.h

```diff
@@ -351,7 +352,30 @@ ListRingBuffer::get_index_( const delay d ) const
   assert( ( size_t ) idx < buffer_.size() );
   return idx;
 }
-}
+
+
+template < unsigned int num_values >
+class MultiValueRingBuffer
+{
+public:
+  MultiValueRingBuffer();
+
+  void add_value( const long idx0, const long idx1, const double val );
+
+  std::array< double, num_values >& get_values( const long idx0 );
```

Comment:
Rename `idx0` in a way consistent with the above.

Reply:
Done.